tests/test_agent_switches.py

```
@@ -0,0 +1,111 @@
+"""המתג של נדב מגיע גם לענן.
+
+**בלי `yaml`.** הוא אינו מותקן ב-CI, והטסט הזה חייב לרוץ בכל סביבה.
+מה שנבדק הוא מבנה מצומצם — קיום job, ‏`needs`, ומחרוזת בתוך `if` —
+ולזה די בקריאה טקסטואלית. הוספת תלות בשביל טסט מבנה היא מחיר מיותר.
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)